### PR TITLE
[DDW-889] Implement blocks component

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR implemented blocks component for home page.

[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-889)

## Screenshots
Screenshot 2019-10-01 10 42 59

## Testing Checklist
- [x] Go to `Blocks` => `Blocks` story on storybook and check if it matches with wireframe

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: stories/blocks.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    transactions: 5,
+  },
+];
+
+storiesOf('Blocks', module)
+  .addDecorator(story => <ThemeDecorator>{story()}</ThemeDecorator>)
+  .add('Blocks', () => (
+    <div
+      style={{
+        backgroundColor: '#121326',
```
Comment (daniloprates) [inline stories/blocks.stories.tsx]: @DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions. For instance, `#121326` would be `$primary-bg`.

File: source/features/shared/footer/components/Footer.scss [hunk 2]
```diff
@@ -99,8 +99,8 @@
         line-height: 1.6;
 
         span {
-          color: rgba(235, 34, 86, 0.5);
-          font-family: ProximaNova, sans-serif;
+          color: rgba(31, 193, 195, 0.3);
```
Comment (daniloprates) [inline source/features/shared/footer/components/Footer.scss]: @DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions.

File: source/features/shared/footer/components/Footer.scss [hunk 3]
```diff
@@ -152,7 +152,7 @@
               text-decoration: none;
 
               &.emailLink {
-                color: rgba(235, 34, 86, 0.5);
+                color: rgba(31, 193, 195, 0.3);
```
Comment (daniloprates) [inline source/features/shared/footer/components/Footer.scss]: @DeeJayElly could you change this approach to CSS styling? This way you can use SCSS variables, that will be used for the different network versions.

File: stories/blocks.stories.tsx (hunk 1 above)
Comment (DeeJayElly) [reply stories/blocks.stories.tsx]: Fixed

File: source/features/shared/footer/components/Footer.scss (hunk 2 above)
Comment (DeeJayElly) [reply source/features/shared/footer/components/Footer.scss]: Fixed

File: source/features/shared/footer/components/Footer.scss (hunk 3 above)
Comment (DeeJayElly) [reply source/features/shared/footer/components/Footer.scss]: Fixed

**Comment by DeeJayElly** [comment]
> Great job! Left some minor requests.

Fixed

File: source/features/shared/blocks/actions.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import Action from '../../../utils/Action';
```
Comment (yakovkaravelov) [inline source/features/shared/blocks/actions.ts]: Is this needed for blocks component? @DeeJayElly

**Comment by DeeJayElly** [comment]
Added fixes and renamed "shared" to "widgets" folder

**Pull request merged**